src/react-queries/queries/accountManagement.ts

```diff
@@ -0,0 +1,24 @@
+import { createQueryKeys } from '@lukemorales/query-key-factory'
+import { type ChainId, fromAccountId } from '@shapeshiftoss/caip'
+import type { HDWallet } from '@shapeshiftoss/hdwallet-core'
+import { deriveAccountIdsAndMetadata } from 'lib/account/account'
+import { assertGetChainAdapter } from 'lib/utils'
+import { checkAccountHasActivity } from 'state/slices/portfolioSlice/utils'
+
+export const accountManagement = createQueryKeys('accountManagement', {
+  loadAccount: (accountNumber: number, chainId: ChainId, wallet: HDWallet) => ({
```

Comment:
Not sure about the vernacular here, what do we mean by `load`?  All this really does is derive a pubkey (`AccountId` in this case) from accountNumber + ChainId using wallet entropy + derive `hasActivity` from it.

I'd probably rename this to `accountIdWithActivity`